IDEAS/Fluid/PvtCollectors/Validation/PVT1/BaseClasses/LongWaveRadiation.mo

```diff
@@ -0,0 +1,114 @@
+    unit="W/m2",
+    displayUnit="kJ/hr.m²") "Diffuse horizontal irradiation [W/m2]"
+    annotation (Placement(transformation(extent={{-140,24},{-100,64}}),
+        iconTransformation(extent={{-140,24},{-100,64}})));
+
+  Modelica.Blocks.Interfaces.RealInput Eglobh_h(
+    quantity="TotalRadiationCollector",
+    unit="W/m2",
+    displayUnit="kJ/hr.m²") "global horizontal irradiation [W/m2]"
+    annotation (Placement(transformation(extent={{-140,68},{-100,108}}),
+        iconTransformation(extent={{-140,68},{-100,108}})));
+
+  // Constants
+  constant Real sigma = 5.67e-8 "Stefan-Boltzmann constant [W/m²K⁴]";
+  constant Real tiltAngle = 45 * pi / 180;
+  constant Real pi = Modelica.Constants.pi "Pi constant";
+  constant Real epsGro = 0.95 "ground emissivity [-]";
+
+  // Constants for dew point calculation using Buck's equation [Buck, 1981]
+  constant Real aBuck = 243.5 "Buck constant for dew point [°C]";
+  constant Real bBuck = 17.67 "Buck constant for dew point";
+
+  // Intermediate Variables
+  Real Tdew "Dew point temperature [°C]";
+  Real epsSky "Clear sky emissivity [-]";
+
+  // Outputs
+  Modelica.Blocks.Interfaces.RealOutput lonRad(
+    quantity="LongwaveRadiation",
+    unit="W/m2",
+    displayUnit="W/m2") "Longwave radiation [W/m²]"
+    annotation (Placement(transformation(extent={{100,-16},{134,18}}),
+        iconTransformation(extent={{100,-16},{134,18}})));
+
+equation
+   // Calculate Dew Point Temperature (Tdew) using Buck's equation [Buck, 1981]
+  Tdew = (aBuck * (bBuck * (Tamb - 273.15) / (aBuck + (Tamb - 273.15)) + log(rH/100))) /
+         (bBuck - (bBuck * (Tamb - 273.15) / (aBuck + (Tamb - 273.15)) + log(rH/100)));
+
+  // Clear Sky Emissivity (epsEmi)
+  epsSky = 0.711 + 0.56 * (Tdew/100) + 0.73 * (Tdew/100)^2;
+
+  // Calculate Longwave Radiation (lonRad) using Tview and the Stefan-Boltzmann Law
+  lonRad = sigma*Tamb^4*((epsSky*(1 + cos(tiltAngle)) / 2) + (epsGro*(1 - cos(tiltAngle)) / 2));
+
+annotation (
+  defaultComponentName="LongWaveRad",
+  Documentation(info = "<html>
+  <p>
+    Computes longwave radiation exchange between the sky and a tilted surface using ambient meteorological inputs. 
+    The model estimates sky emissivity based on the dew point temperature derived from ambient temperature and relative humidity, 
+    following Buck's equation. It then applies the Stefan–Boltzmann law to calculate the net longwave radiation on a tilted surface, 
+    accounting for both sky and ground contributions.
+  </p>
+
+  <h4>Implementation Notes</h4>
+  <p>
+  This model is used in the validation of unglazed photovoltaic–thermal (PVT) collectors 
+  (<a href=\"modelica://IDEAS.Fluid.PVTCollectors.Validation.PVT1.PVTQuasiDynamicCollectorValidation\"> IDEAS.Fluid.PVTCollectors.Validation.PVT1.PVTQuasiDynamicCollectorValidation</a>) 
+  where direct longwave irradiance measurements are unreliable or unavailable. It uses a fixed tilt angle of 45° and assumes a ground emissivity of 0.95. 
```

Comment:
Why fix this in the model and not overwrite this in `PVTQuasiDynamicCollectorValidation`?